docs/New-ExternalHelp.md

```diff
@@ -90,57 +82,57 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Override existing files.
+Indicates that this cmdlet overwrites an existing file that has the same name.
 
 
 ```yaml
 Type: SwitchParameter
 Parameter Sets: (All)
-Aliases: 
+Aliases:
 
 Required: False
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -Path
-Path to markdown files or directories.
+Specifies an array of paths of markdown files or folders. This cmdlet creates external help based on these files and folders.
 
 
 ```yaml
 Type: String[]
 Parameter Sets: (All)
-Aliases: 
+Aliases:
 
 Required: True
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: True (ByPropertyName, ByValue)
 Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
 ## INPUTS
 
 ### String[]
-You can pipe a collection of paths to this cmdlet.
+You can pipe an array of paths to this cmdlet.
 ## OUTPUTS
 
 ### System.IO.FileInfo[]
-This cmdlet returns a FileInfo[] object for created files.
+This cmdlet returns a **FileInfo[]** object for created files.
 ## NOTES
 
 ## RELATED LINKS
 
-[PowerShell V2 External MAML Help](https://blogs.msdn.microsoft.com/powershell/2008/12/24/powershell-v2-external-maml-help/)
-
 [Online Version:](https://github.com/PowerShell/platyPS/blob/master/docs/New-ExternalHelp.md)
 
+[New-ExternalHelpCab](New-ExternalHelpCab.md)
 
+[Schema](https://github.com/PowerShell/platyPS/blob/master/platyPS.schema.md)
 
+[PowerShell V2 External MAML Help](https://blogs.msdn.microsoft.com/powershell/2008/12/24/powershell-v2-external-maml-help/)
 
-
-
+[Get-Help](https://msdn.microsoft.com/en-us/library/dd878343.aspx)
```

Comment:
We should be able to leverage [markdown shortcut reference links](http://spec.commonmark.org/0.25/#shortcut-reference-link) to avoid duplicates in the document and in `RELATED LINKS` section.

I opened #138 for that